i18n/Messages_kr.properties

```diff
@@ -8,8 +8,8 @@ label.player-count=플레이어 수
 label.discard-area=카드 버리기
 label.blanked=무효
 label.cursed-hoard=Cursed Hoard
```

Comment:
Could you translate this label?

Reply:
Thanks for the feedback.
However, I didn't find where the word "Cursed Hoard" is used, could you please point me to where it is used?

Reply:
Good point. Turns out the "label.cursed-hoard" is not currently used.